recipes/ODENet/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set name = "ODENet" %}
+{% set version = "0.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }} 
+
+source:
+  url: https://github.com/uqyge/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: d060a64004b12fbb61eff60618f8afd5c843f4dc4f5289b5df195269e69a23ab
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

I think this recipe can be ``noarch``
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
  noarch: python
```

---

thx so much, I am committing the suggestion. It's my first time doing conda packages, could you point me to some material about the reason for doing noarch?